pkg/kubelet/sysctl/runtime.go

```diff
@@ -34,16 +35,17 @@ const (
 
 // TODO: The admission logic in this file is runtime-dependent. It should be
 // changed to be generic and CRI-compatible.
+// RuntimeAdmitHandler returns lifecycle.PodAdmitResult
 
-type runtimeAdmitHandler struct {
+type RuntimeAdmitHandler struct {
 	result lifecycle.PodAdmitResult
 }
 
-var _ lifecycle.PodAdmitHandler = &runtimeAdmitHandler{}
+var _ lifecycle.PodAdmitHandler = &RuntimeAdmitHandler{}
 
-// NewRuntimeAdmitHandler returns a sysctlRuntimeAdmitHandler which checks whether
+// newRuntimeAdmitHandler returns a sysctlRuntimeAdmitHandler which checks whether
 // the given runtime support sysctls.
-func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, error) {
+func newRuntimeAdmitHandler(runtime container.Runtime) (*RuntimeAdmitHandler, error) {
```

Comment:
Unfortunately, we're not going to be able to make `newRuntimeAdmitHandler` unexported, as its used in a different package: https://sourcegraph.com/github.com/kubernetes/kubernetes@master/-/blob/pkg/kubelet/kubelet.go#L844:33

However, I believe that making `RuntimeAdmitHandler` an exported type should be sufficient to silence the linter.

However, you may see another stuttering error, which I think we could resolve by renaming this type from `RuntimeAdmitHandler` to just `AdmitHandler`.

As a reminder, when we rename the struct type, we also need to rename everywhere we use the struct. For example, check out line 62 in this file... currently, it is using `runtimeAdmitHandler`, even though we updated the struct type name to be `RuntimeAdmitHandler`. As a result, go is failing to build the package.

Reply:
as a result,is it that rename runtimeAdmitHandler to AdmitHandler in everywhere ,is that so?